docs/en/commands/element/click.md

```diff
@@ -0,0 +1,72 @@
+# Click
+
+Retrieve the server’s current status.
+
+## Example Usage
+
+```java
+// Java
+MobileElement el = driver.findElementByAccessibilityId("SomeId");
+el.click();
+```
+```python
+# Python 
+el = self.driver.find_element_by_accessibility_id('SomeId')
+el.click();
+```
+```javascript
+// Javascript
+// webdriver.io
+driver.click('#SomeId');
+
+// wd (async/await)
+var element = await driver.elementByAccessibilityId('id', 'SomeId');
```

Comment:
come on, `var`? :-)

Reply:
Yeah, I did it because it's the lowest common denominator. But I suppose if we're using async/await, we can use let/const as well.